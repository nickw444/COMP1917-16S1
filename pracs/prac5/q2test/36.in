36
-6 -10 0 5 -4 7 7 10 -10 8 1 -8 -10 -1 -9 -7 6 -9 -6 -8 10 10 -2 -6 -9 -10 5 0 -6 0 1 10 0 -2 5 -9
