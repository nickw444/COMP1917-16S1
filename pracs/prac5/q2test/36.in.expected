average of array: -1.305556
